docs/AboutUs.adoc

```diff
@@ -35,7 +35,7 @@ image::zhicaizack.png[width="150", align="left"]
 {empty}[https://github.com/zhicaizack[github]] [<<zhicaizack#, portfolio>>]
 
 Role: Developer +
```

Comment:
Remember to change from `Developer` to your proper role name :)

Reply:
changed as requested 👍